doc/apimanagerbackup-reference.md

```diff
@@ -0,0 +1,111 @@
+  *  When the location of System's FileStorage is in a PersistentVolumeClaim (PVC)
+  * **CURRENTLY UNSUPPORTED** When the location of System's FileStorage is in a S3 API-compatible storage
+
+## Data that is not backed up
+
+Backups of the external databases used by 3scale is not part of the
+3scale-operator functionality and has to be performed by the user appropriately
+
+## APIManagerBackup
+
+| **json/yaml field**| **Type** | **Required** | **Description** |
+| --- | --- | --- | --- |
+| `spec` | [APIManagerBackupSpec](#APIManagerBackupSpec) | Yes | The specfication for APIManagerBackup custom resource |
+| `status` | [APIManagerBackupStatusSpec](#APIManagerBackupStatusSpec) | No | The status of APIManagerBackup custom resource |
+
+### APIManagerBackupSpec
+
+| **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
+| --- | --- | --- | --- | --- | --- |
+| `apiManagerName` | string | No | Name of the APIManager deployed in the same namespace as the deployed APIManagerBackup | Name of the APIManager to backup |
+| `backupSource` | [APIManagerBackupSourceSpec](#APIManagerBackupSourceSpec) | Yes | See [APIManagerBackupSourceSpec](#APIManagerBackupSourceSpec) | Configuration related to where the backup is performed |
+
+### APIManagerBackupSourceSpec
+
+This section controls where APIManager's backup is to be stored.
+**One of the fields is mandatory to be set. Only one of the fields can be set. The fields are mutually exclusive.**
+
+| **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
+| --- | --- | --- | --- | --- |
+| `persistentVolumeClaim` | [PersistentVolumeClaimBackupSource](#PersistentVolumeClaimBackupSource) | No | nil | APIManager backup destination in PVC |
+| `simpleStorageService` | [APIManagerBackupS3Source](#APIManagerBackupS3Source) | No | nil | **CURRENTLY UNSUPPORTED** APIManager backup destination in S3 api-compatible service |
+
+### PersistentVolumeClaimBackupSource
+
+There are two main ways to provide a PersistentVolumeClaim for the backup:
+* Providing the volume name of an already existing Kubernetes PersistentVolume
+  through the usage of the `volumeName` field. The already existing Kubernetes
+  PersistentVolume has to be appropriately sized to contain
+  all [data that is backed up](#data-that-is-backed-up)
+* Providing the desired size of the PersistentVolumeClaim through the
+  `resources` field. In this case the `storageClass` field
+  can be also set to specify what StorageClass will be used for the
+  PersistentVolumeClaim if desired. Make sure enough resources are defined
+  to contain all [data that is backed up](#data-that-is-backed-up)
+
+| **json/yaml field**| **Type** | **Required** | **Default value** | **Description** |
+| --- | --- | --- | --- | --- |
+| `resources` | [PersistentVolumeClaimResourcesSpec](#PersistentVolumeClaimResourcesSpec) | No | See [PersistentVolumeClaimResourcesSpec](#PersistentVolumeClaimResourcesSpec) | 
```

Comment:
From backup CRD, `resources` are required?

Reply:
Yes.

When you create a PVC, if you do not provide an existing PV name you have to specify the minimum size that you want. See PersistentVolumeClaimResourcesSpec section with details on what you can configure

Reply:
`Required` field value :eyes:

Reply:
:+1: I'll add the optional annotation

Reply:
Added